lib/compiler/passes/generate-bytecode.js

```diff
@@ -551,11 +572,11 @@ function generateBytecode(ast) {
     },
 
     semantic_and(node, context) {
-      return buildSemanticPredicate(node.code, false, context);
+      return buildSemanticPredicate(node, false, context);
```

Comment:
"buildSemanticPredicate" could mean ***literally anything***

Reply:
Well, that name already used and I'm not mix PR changes with random changes not related to the main goal of my PRs

Reply:
okay, that's fair.  withdrawn